mypy.ini

```diff
@@ -0,0 +1,5 @@
+[mypy]
+python_version = 3.9
+warn_return_any = True
+warn_unused_configs = True
+ignore_missing_imports = True
```

Comment:
Added a `mypy` configuration while working on this so that we can work with proper extension that adds static typing to the project. Feel free to change as you want.